VagrantProvision.sh

```diff
@@ -168,6 +179,7 @@ if  ! dpkg -l | grep --quiet google-chrome-stable; then
     if [ ! -f google-chrome-stable_current_amd64.deb ]; then
       wget --quiet https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
     fi
+    sudo apt-get -f -y -q install
```

Comment:
I found that it reduced the amount of warnings coming from installing chrome from a `.deb` file while all of the prereqs are installed from `apt-get`.  Still not completely green but a good step in the right direction.